guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -261,6 +261,31 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
     };
 
     /**
+     * Returns whether the current user can change/set all user attributes for
+     * the user being edited within the given data source, regardless of
+     * whether those attributes are already explicitly associated with that
+     * user.
+     *
+     * @param {String} [dataSource]
```

Comment:
Fair enough, and true of all the past similar `can*()` functions, too. I was going to argue that this is the pattern used by similar things here, but that's silly. It's wrong and we should stop wronging it.